pkg/apis/apps/types.go

```diff
@@ -82,6 +97,16 @@ type StatefulSetSpec struct {
 	// pattern: pod-specific-string.serviceName.default.svc.cluster.local
 	// where "pod-specific-string" is managed by the StatefulSet controller.
 	ServiceName string
+
+	// PodCreationPolicy controls how pods are created during initial scale up,
+	// when replacing pods on nodes, or when scaling down. The default policy is
+	// `OrderedReady`, where pods are created in increasing order (pod-0, then
+	// pod-1, etc) and the controller will wait until each pod is ready before
+	// continuing. When scaling down, the pods are removed in the opposite order.
+	// The alternative policy is `Parallel` which will create pods in parallel
+	// to match the desired scale without waiting, and on scale down will delete
+	// all pods at once.
+	PodCreationPolicy PodCreationPolicyType
```

Comment:
This value applies beyond updates and to all operations.  See earlier comment on the issue (if they're still visible).  Because of that it's a property of the stateful set, not of the strategy for updates